jax/random.py

```diff
@@ -781,7 +782,8 @@ def laplace(key, shape=(), dtype=onp.float64):
 @partial(jit, static_argnums=(1, 2))
 def _laplace(key, shape, dtype):
   _check_shape("laplace", shape)
-  u = uniform(key, shape, dtype, minval=-1., maxval=1.)
+  u = uniform(
+      key, shape, dtype, minval=-1. + np.finfo(np.float32).epsneg, maxval=1.)
```

Comment:
I'm not an expert in these things, but I wonder if something like `onp.nextafter(onp.array(-1, dtype), 1)` could be a useful line here, as it would avoid hardcoding the float32 dtype. I doubt it would matter in practice though. What do you think?

Reply:
Sorry about that, I forgot to pass dtype to np.finfo... I've amended the code to pass it along.